src/sidebar/services/test/frame-sync-test.js

```diff
@@ -216,42 +210,48 @@ describe('FrameSyncService', () => {
       frameSync.connect();
     });
 
-    it('sends a "loadAnnotations" message to the frame', () => {
+    it('sends a "loadAnnotations" message to the frame', async () => {
+      await connectGuest();
```

Comment:
Several tests for messages sent to guests did not actually simulate a guest connecting before checking that the message was sent. This didn't matter before because events were sent on the bridge whether or not any guests were connected. Now there will only be `PortRPC` connections if a guest is actually connected, so we have to simulate a guest connecting first.